framework/src/application/application.ts

```diff
@@ -30,21 +30,24 @@ import {
 import { Contexter } from '@liskhq/lisk-chain';
 import { KVStore } from '@liskhq/lisk-db';
 import { getNetworkIdentifier } from '@liskhq/lisk-cryptography';
-import { validator as liskValidator } from '@liskhq/lisk-validator';
+import { validator } from '@liskhq/lisk-validator';
 import * as _ from 'lodash';
 import { systemDirs } from './system_dirs';
 import { Controller, ModulesOptions } from '../controller/controller';
 import { version } from '../version';
-import * as validator from './validator';
-import * as configurator from './default_configurator';
-import { genesisBlockSchema, constantsSchema } from './schema';
+import {
+	genesisBlockSchema,
+	constantsSchema,
+	mergeDeep,
+	applicationConfigSchema,
+} from './schema';
 import { ApplicationState } from './application_state';
 import { Network } from './network';
 import { Node } from './node';
 import { InMemoryChannel } from '../controller/channels';
 import { Logger, createLogger } from './logger';
 
-import { DuplicateAppInstanceError } from '../errors';
+import { DuplicateAppInstanceError, SchemaValidationError } from '../errors';
```

Comment:
Can we replace this `SchemaValidationError` with `lisk-validator` error `LiskValidationError`?